src/index.js

```diff
@@ -146,9 +152,9 @@ export default class Pipeliner {
 
   /**
    * Runs the specified pipeline, passing the arguments to each task.
-   * @param  {string}    pipeline the pipeline to run
+   * @param  {string}    pipeline The name of the pipeline to run
    * @param  {...object} args     Arguments to pass to the pipeline tasks
-   * @return {Promise}             A promise that is executed when the pipeline completes.
+   * @return {Promise}            A promise that is fulfilled when the pipeline completes; it is rejected if any task in the pipeline fails (throws an error or returns a promise that is rejected)
```

Comment:
If this is true, then we need to change line 160 to use Promise.try or Promise.method rather than Promise.resolve. Promise.resolve will always a resolved promise, whether or not the wrapping method throws an exception.